src/algopy_testing/op.py

```diff
@@ -847,6 +847,62 @@ def app_opted_in(
     return app_id in account_data.opted_apps
 
 
+class _AcctParamsGet:
+    def __getattr__(
+        self, name: str
+    ) -> typing.Callable[[algopy.Account | algopy.UInt64 | int], tuple[typing.Any, bool]]:
+        def get_account_param(a: algopy.Account | algopy.UInt64 | int) -> tuple[typing.Any, bool]:
+            import algopy
+
+            from algopy_testing.context import get_test_context
+
+            context = get_test_context()
+            if not context:
+                raise ValueError(
+                    "Test context is not initialized! Use `with algopy_testing_context()` to "
+                    "access the context manager."
+                )
+
+            active_txn = context.get_active_transaction()
+            if not active_txn:
+                raise ValueError("No active transaction found to reference asset")
+
+            account_data = None
+
+            if isinstance(a, (algopy.Account)):
+                account_data = context.get_account(str(a))
```

Comment:
Rather than using the `str` builtin and relying on the `__str__` implementation we should have an explicit `public_key` property on the `Account` type. 

To make that work with mypy the isinstance check here should be for `algopy_testing.models.account.Account`

Reply:
Resolved, feel free to reopen if you think something else is missing